### PR TITLE
Added Missing Translation for selectMeshFor refs #2134

### PR DESCRIPTION
**Pull request by dartajax**
refs #2134

### CODE REVIEW

**Comment by jrjohnson** [comment]
Deploy preview ready!

Built with commit d5c68e6df3e5c1ff8bb109d74f62129ba4a53b27

http://deploy-preview-2199.ilios-frontend.netlify.com

File: app/templates/components/mesh-manager.hbs [hunk 1]
```diff
@@ -1,5 +1,7 @@
 {{#if targetItemTitle}}
-  <h2>{{t 'selectMeshFor' title=targetItemTitle}}</h2>
+ <span class='specific-title'>
```
Comment (stopfstedt) [inline app/templates/components/mesh-manager.hbs]: what gives with the markup change?

Comment (dartajax) [inline app/templates/components/mesh-manager.hbs]: Well <h3> was strangely even larger - I was trying to save some space.  Does it need to be reverted to <h2>?

Comment (stopfstedt) [inline app/templates/components/mesh-manager.hbs]: yes, please revert to `<h2>`. for now.

Comment (dartajax) [inline app/templates/components/mesh-manager.hbs]: ok I will do that

Comment (dartajax) [inline app/templates/components/mesh-manager.hbs]: it might be tomorrow though

**Comment by saschaben** [comment]
fixes #2134

**Pull request merged**